invenio_communities/bundles.py

```diff
@@ -42,7 +42,7 @@
 
 css = NpmBundle(
     'scss/invenio_communities/communities.scss',
-    filters='scss, cleancss',
+    filters='scss,cleancssurl',
```

Comment:
Is this needed after https://github.com/miracle2k/webassets/pull/486 was merged? Or just using updated `webassets` is enough?